# üß™üî¨ Welcome to my GitHub! üíª‚àö

![SPT](https://github.com/alexSysBio/alexSysBio/blob/main/particle_tracking.gif)

---

## About me üë§
I am a **Research Specialist** in the **Jacobs-Wagner Lab** at **Stanford University**. My research integrates **high-content imaging** with **big-data analysis** to understand how cellular functions, homeostasis, and phenotypic variability emerge from the physical interactions between biologically significant components, the scaling of growth constituents, and the biosynthetic gating of cell cycle processes. 

---

## Open science ‚öõÔ∏è
I am guided by my vision for an openly accessible, ethical science that addresses contemporary biomedical and environmental challenges while expanding our quantitative understanding of fundamental biological processes.

### Big-data contributions ###

<div style="display: flex; flex-direction: column; align-items: flex-start;">
    
1. [BioImage Archive](https://www.ebi.ac.uk/biostudies/bioimages/studies/S-BIAD1658)
2. [BioImage Archive](https://www.ebi.ac.uk/biostudies/bioimages/studies/S-BIAD1350)

</div>

---

## Publications üìö
[Google Scholar](https://scholar.google.com/citations?user=sxnPVMcAAAAJ&hl=en) | 
[ORCID](https://orcid.org/0000-0002-6363-804X)


### Most representative publications ###
(üåü = first-author contribution)

<div style="display: flex; flex-direction: column; align-items: flex-start;">
    
1. üåü **A. Papagiannakis**, Q. Yu, S.K. Govers, W-H. Lin, N.S. Wingreen, and C. Jacobs-Wagner (2025).
   *Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli.* 
   [eLife](https://doi.org/10.7554/eLife.104276.3), doi: [10.7554/eLife.104276.3](https://doi.org/10.7554/eLife.104276.3)  

2. üåü J. M√§kel√§, **A. Papagiannakis**, W-H. Lin, M.C. Lanz, S. Glenn, M. Swaffer, G.K. Marinov, J.M. Skotheim, and C. Jacobs-Wagner (2024).
   *Genome concentration limits cell growth and modulates proteome composition in Escherichia coli.* 
   [eLife](https://elifesciences.org/articles/97465), 13:RP97465  

3. üåü **A. Papagiannakis**, B. Niebel, E.C. Wit, M. Heinemann (2017).
   *Autonomous metabolic oscillations robustly gate the early and the late cell cycle.* 
   [Molecular Cell](https://pubmed.ncbi.nlm.nih.gov/27989441/), 65(2): 285-295.  

4. A.D. Sanchez, T.C. Branon, L.E. Cote, **A. Papagiannakis**, X. Liang, M.A. Pickett, K. Shen, C. Jacobs-Wagner, A.Y. Ting, J.L. Feldman (2021). 
   *Proximity labeling reveals non-centrosomal microtubule-organizing center components required for microtubule growth and localization.* 
   [Current Biology](https://pubmed.ncbi.nlm.nih.gov/34242576/), 31(16): 3586-3600.e11.  

5. üåü **A. Papagiannakis**, J.J. de Jonge, Z. Zhang, M. Heinemann (2017).
   *Quantitative characterization of the auxin-inducible degron: a guide for dynamic protein depletion in single yeast cells.* 
   [Scientific Reports](https://www.nature.com/articles/s41598-017-04791-6), 7: 4704.  

</div>

---


## Connect with me ü§ù
Feel free to reach out on [LinkedIn](https://www.linkedin.com/in/alex-papagiannakis-singlecells/) or follow my work on GitHub.

---

## Public repositories üå≥
[omePyfun](https://github.com/alexSysBio/omePyfun) | 
[DiffractionPySim](https://github.com/alexSysBio/DiffractionPySim) | 
[PySimuNS](https://github.com/alexSysBio/PySimuNS) | 
[NDtwoPy](https://github.com/alexSysBio/NDtwoPy) | 
[sptPy](https://github.com/alexSysBio/sptPy) | 
[PyMedialAxis](https://github.com/alexSysBio/PyMedialAxis) | 
[UnBack](https://github.com/alexSysBio/UnBack) | 
[ManuCure](https://github.com/alexSysBio/ManuCure) | 
[SuperMaskClass](https://github.com/alexSysBio/SuperMaskClass) | 
[PyntensityProfile](https://github.com/alexSysBio/PyntensityProfile) | 
[PycellMask](https://github.com/alexSysBio/PycellMask) | 
[UnDrift](https://github.com/alexSysBio/UnDrift) | 

---

Thank you for visiting my GitHub profile! üê¨

<img src="https://github.com/alexSysBio/alexSysBio/blob/main/IMG_8688.jpg" alt="Project Overview" width="50%"/>


